File: app/src/main/java/oleg/osipenko/mai/data/repository/specification/StaticContentSpecification.java
package oleg.osipenko.mai.data.repository.specification;

/**
 * Created by olegosipenko on 13.09.15.
 */
public class StaticContentSpecification implements RequestSpecification<String> {
    private String item;

    public StaticContentSpecification(String item) {
        this.item = item;
    }

    @Override
    public boolean specified(String s)  {
        return s.equals(item);
    }

    public String getItem() {
        return item;
    }
}

File: app/src/main/java/oleg/osipenko/mai/data/repository/specification/StaticContentSpecification.kt
package oleg.osipenko.mai.data.repository.specification

import oleg.osipenko.mai.data.repository.RequestSpecification

/**
 * Created by olegosipenko on 05.09.15.
 */
public class StaticContentSpecification : RequestSpecification<String> {
    override fun specified(s: String): Boolean {
        return false
    }
}
